src/jobs/run.yml

```diff
@@ -7,19 +7,24 @@ parameters:
     type: string
 
   formatter:
-    description: "Formatter to use for output. Select `junit` for CircleCI test reporting. You can also supply a custom formatter."
+    description: "Formatter to use for output seen in the step UI. You can also supply a custom formatter."
     type: string
     default: "tap"
 
+  report_formatter:
+    description: "Formatter to use for test report output. Select `junit` for CircleCI test reporting. You can also supply a custom formatter."
+    type: string
+    default: "junit"
+
   timing:
     description: "Add timing information to the tests. Recommended for CircleCI test reporting."
     type: boolean
-    default: false
+    default: true
```

Comment:
Note that `true` and `false` values do not align with the value that the `run-bats.sh` script is looking for, which is a bit confusing.
According to the [CircleCI docs](https://circleci.com/docs/reusing-config/#boolean) a boolean value _may_ be passed in as a `1` or `0` but it doesn't mandate that it always will do (the CircleCI docs don't specify what it will do, merely what it may do in addition to potentially passing in the values unmodified).

In order to make the script cope with all variations that the CircleCI spec says it can do, it'd have to cope with true, yes, y etc.

As I'm kinda new to CircleCI, I don't know if that's a flaw in this orb's script or a flaw in the CircleCI documentation, but _one of them_ isn't right...

Reply:
I can confirm the YAML parser used by CircleCI will always return a 1 for truthy statements

Reply:
OK; sounds like the documentation is lacking; that translation from "truthy" to "1" should be documented as a "will happen" not a "may (but might not) happen".